### PR TITLE
Replace express-request-id

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
         "functions": "never"
       }
     ],
-    "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/*.test.js", "**/*.test.ts"] }],
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
```
Comment (psoleckimoj) [inline .eslintrc.json:96]: Couldn't work out why it only complained about the type for uuid and no other modules?

File: server/@types/bapv.d.ts [hunk 2]
```diff
@@ -1,5 +1,13 @@
 import { InmateDetail, VisitBalances } from '../data/prisonApiTypes'
 
+declare global {
```
Comment (psoleckimoj) [inline server/@types/bapv.d.ts:3]: Needed to allow adding `id` to the request object.

**Comment by psoleckimoj** [comment]
@andrewrlee this is a quick draft of trying to pull the code into our own repo. Not tested, just thought I'd get your thoughts 🙂

File: server/@types/bapv.d.ts (hunk 2 above)
Comment (andrewrlee) [reply server/@types/bapv.d.ts:3]: would it be better to add the type override [here](https://github.com/ministryofjustice/book-a-prison-visit-staff-ui/blob/9b21c7f7076e16184404856b11404881af2cc90a/server/%40types/express/index.d.ts#L19)

File: .eslintrc.json (hunk 1 above)
Comment (andrewrlee) [reply .eslintrc.json:96]: not sure this is right? Doesn't the change says that it shouldn't complain if you use a dev import in any code including prod? 

I think you just need to install `uuid` explicitly as a prod dependency?

File: server/@types/bapv.d.ts (hunk 2 above)
Comment (psoleckimoj) [reply server/@types/bapv.d.ts]: Done 👍

**Comment by psoleckimoj** [comment]
@andrewrlee I've made all the changes and all seems to be working well on dev. @tpmcgowan this sorts one of the issues with updating the `express-request-id` module by removing it 😁

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**